packages/react-components/src/Concepts/Icons/ReactIconGrid.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { FluentIconsProps } from '@fluentui/react-icons';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import * as ReactIcons from '@fluentui/react-icons';
+import { Input } from '@fluentui/react-input';
+import { makeStyles } from '@fluentui/react-make-styles';
+
+const useStyles = makeStyles({
+  grid: theme => ({
+    width: '100%',
+    marginLeft: 'auto',
+    boxSizing: 'border-box',
+    marginRight: 'auto',
+    display: 'block',
+    paddingLeft: '16px',
+    paddingRight: '16px',
+    paddingTop: '10px',
+    fontFamily: 'Arial',
```

Comment:
This component should inherit its font family from the parent rather than setting its own that might not match. If that's causing some kind of issue, we can find a different workaround.
```suggestion
```